### DISCUSSION TITLE
Add an AllowedContentTypeAliases to the Content Delivery API

### DISCUSSION THREAD

**Discussion by shropgov-neil** [umbraco/Umbraco-CMS#15806] [Features and ideas]
The docs for the Content Delivery API refer to the "DisallowedContentTypeAliases", where you can specify which content types to exclude from the API.

It would be complementary to offer an "AllowedContentTypeAliases" setting, for situations where zero-trust is preferred, and newly createdcontent types are not included by default, and only if explicitly added to the list.

**Comment by AndyButland** [comment]
Resolved via https://github.com/umbraco/Umbraco-CMS/pull/21111 for 17.1.
